calligraphy/src/main/java/uk/co/chrisjenx/calligraphy/CalligraphyFactory.java

```diff
@@ -237,7 +237,9 @@ private ToolbarLayoutListener(final CalligraphyFactory calligraphyFactory,
                 }
             }
             removeSelf(toolbar);
-            toolbar.setSubtitle(originalSubTitle);
+            if (TextUtils.equals(toolbar.getSubtitle(), BLANK)) {
```

Comment:
not sure, but could you test for strict object equality using `==` here? as of now, it's possible for the user to set the subtitle to `" "`, which would get blown away by this block. but it's impossible for the user to set the subtitle to the string object referenced by `BLANK`, because this is a private class.

using strict equality would of course depend on `getSubtitle()` always returning the exact object that you previously passed to `setSubtitle()`, and not another, merely `equals` object, so it wouldnt reliably work across say lifecycle restarts, or anything else where the view is getting rebuilt from saved state.